src/include/sof/drivers/sai.h

```diff
@@ -18,6 +18,8 @@
 
 #ifdef CONFIG_IMX8M
 #define SAI_OFS		8
```

Comment:
can you add here
```
#if defined(CONFIG_IMX8M) ||  defined(CONFIG_IMX8ULP)
#define   SAI_OF 8
#else    
#define SAI_OFS 0
```

https://stackoverflow.com/questions/9682593/how-to-use-ifdef-with-an-or-condition